ansible/roles/kafka/tasks/deploy.yml

```diff
@@ -15,7 +15,7 @@
       - "{{ zookeeper.port }}:2181"
 
 - name: wait until the Zookeeper in this host is up and running
-  action: shell echo ruok | nc -w 3 -i 1 {{ inventory_hostname }} {{ zookeeper.port }}
+  action: shell (echo ruok; sleep 1) | nc {{ inventory_hostname }} {{ zookeeper.port }}
```

Comment:
On CentOS, if idle timeout happens according to `-i 1` option, it displays the timeout log.
`Ncat: Idle timeout expired (1000 ms).`

I am not sure about other version of CentOS, but it did not work on CentOS 7